src/backend/api/library.ts

```diff
@@ -38,8 +30,16 @@ export const importGame = async (args: ImportGameArgs) =>
   ipcRenderer.invoke('importGame', args)
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const handleSetGameStatus = (callback: any) =>
+export const handleSetGameStatus = (callback: any) => {
   ipcRenderer.on('setGameStatus', callback)
+  return () => {
+    ipcRenderer.removeListener('setGameStatus', callback)
+  }
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// export const setGameStatusRemoveListener = (onGameStatusUpdate: any) => {
+//   ipcRenderer.removeListener('setGameStatus', onGameStatusUpdate)
+// }
```

Comment:
If this is not needed anymore maybe remove it?

Reply:
done